llvm/include/llvm/SYCLLowerIR/DeviceConfigFile.td

```diff
@@ -86,6 +86,7 @@ def AspectExt_oneapi_cuda_cluster_group : Aspect<"ext_oneapi_cuda_cluster_group"
 def AspectExt_intel_fpga_task_sequence : Aspect<"ext_intel_fpga_task_sequence">;
 def AspectExt_oneapi_atomic16 : Aspect<"ext_oneapi_atomic16">;
 def AspectExt_oneapi_virtual_functions : Aspect<"ext_oneapi_virtual_functions">;
+def AspectExt_intel_spill_mem_size : Aspect<"ext_intel_spill_mem_size">;
```

Comment:
The name is very specific. It should probably cover the whole set of traits in the extension. I've put the proper name in the spec.